chat-sdk/src/main/java/com/amazon/connect/chat/sdk/network/WebSocketManager.kt

```diff
@@ -479,6 +484,26 @@ class WebSocketManagerImpl @Inject constructor(
         }
     }
 
+    private fun handleCommonChatEvent(innerJson: JSONObject, rawData: String): TranscriptItem {
```

Comment:
Lots of duplicate logic as https://github.com/amazon-connect/amazon-connect-chat-android/blob/8657130358ffc038db4a722ab30da3f9f5776d80/chat-sdk/src/main/java/com/amazon/connect/chat/sdk/network/WebSocketManager.kt#L424

Could we simplify this so we're only creating the Event object once?